bindata/istio-csr/certificate.yaml

```diff
@@ -0,0 +1,32 @@
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: istiod
+  namespace: istio-system
```

Comment:
would this cause operator to get stuck on the reconcile if `istio-system` namespace is not present on the cluster?

Reply:
`namespace` in the manifests is updated by the controller with the user defined value, and the config field for the same is mandatory.
Yes, controller will be in reconcile loop, if the configured namespace does not exist, controller will not create the namespace.

Reply:
you might want to remove the namespace in the jsonnet / wherever you end up allowing patching these upstream istio-csr static manifests.

get your point, logically this'll be anyway overriden in code so shouldn't be a big problem irrespective.